protocol/chainlib/chainproxy/rpcInterfaceMessages/restMessage_test.go

```diff
@@ -2,20 +2,21 @@ package rpcInterfaceMessages
 
 import (
 	"testing"
+
+	"github.com/stretchr/testify/require"
 )
 
 func TestRestMessage(t *testing.T) {
 	// Test GetParams method
 	restMessage := RestMessage{
-		Path: "eth_getTransactionByHash",
-		Msg:  []byte{1, 2, 3, 4, 5},
+		Path:     "eth_getTransactionByHash",
```

Comment:
eth has no rest messages so it might be confusing ethereum uses only jsonrpc

Reply:
Fixed: ae1eabb9451c17943a95cdbe083634adc9dfb073